lib/client-assets.php

```diff
@@ -282,9 +282,17 @@ function gutenberg_register_scripts_and_styles() {
 			"\n",
 			array(
 				'( function() {',
-				'	if ( wp && wp.components && wp.serverSideRender && ! wp.components.ServerSideRender ) {',
-				'		wp.components.ServerSideRender = wp.serverSideRender;',
-				'	};',
+				'	if ( wp && wp.components && wp.serverSideRender ) {',
```

Comment:
I'm a bit worried that this might not work in all cases. What if someone has a block which imports `ServerSideRender` from `wp.components` but never references `wp-server-side-render` as a dependency. It's a tough problem. I see that we expose `ServerSideRender` from `wp.editor`, so maybe we should attach this polyfill there without any guard clauses? Or even we could include it inside `@wordpress/editor` which is less likely to be used outside of WordPress.